### PR TITLE
Cleanup README

### PR DESCRIPTION
**Pull request by btbytes**
* make markdown lint happy
* fix url linking
* small typos and other rearrange the deck chair activities

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,4 +5,5 @@ twitter-archive
 .*
 *.json
 !package.json
-!package-lock.json
\ No newline at end of file
+!package-lock.json
+bun.lockb
```
Comment (sankalp1999) [inline .gitignore:9]: what does this do

File: README.md [hunk 2]
```diff
@@ -1,148 +1,133 @@
...
 or if too many pictures are missing, please re-run the script.
 
-Customer support: early access users contact me [here](https://twitter.com/dejavucoder)
-
-P.S
+Customer support: early access users contact me [https://twitter.com/dejavucoder](https://twitter.com/dejavucoder)
 
-If you have already cloned, just git pull once to fetch latest updates.
-`git pull`
+NOTE: If you have already cloned, run `git pull` to fetch latest updates.
```
Comment (sankalp1999) [inline README.md:63]: not sure if I should keep this line in general or not. I have added it to setup.sh

Comment (sankalp1999) [reply README.md:63]: yet to push that change.

File: README.md [hunk 3]
```diff
@@ -1,148 +1,133 @@
...
 
 ## A tool to visualize your Twitter network and direct messaging history
 
 ## Features
 
-1. Make a Twitter Circle visualization for upto 200 users
+1. Make a Twitter Circle visualization for up to 200 users
 2. Check leaderboard based on combined weights of *all* your mentions of other users and *all* direct messages
-3. Check DM stats (purely messages) like messages sent/recieved per user, total messages, last message with them
+3. Check DM stats (purely messages) like messages sent/received per user, total messages, last message with them
```
Comment (sankalp1999) [inline README.md:11]: you got me ahaha. this typo caused bugs in my code too

**Comment by btbytes** [comment]
I use bun.sh instead of npm. This ignores the lock file gen by that package
manager

**Comment by sankalp1999** [comment]
image
LGTM, thanks a lot for the help.

**Review by sankalp1999 — APPROVED** [review]

**Pull request merged**